WordPress/src/main/res/values/strings.xml

```diff
@@ -2234,6 +2234,8 @@
     <string name="support_push_notification_title">WordPress</string>
     <string name="support_push_notification_message">New message from \'Help &amp; Support\'</string>
     <string name="contact_fragment_title" translatable="false" tools:ignore="UnusedResources">@string/contact_support</string>
+    <string name="support_forum_title">Community forums</string>
+    <string name="support_forum_caption">Get help from our group of volunteers</string>
```

Comment:
There is `.` at the end of the sentence on the design file and the issue screenshot.
> Get help from our group of volunteers.